web/components/navbar/index.js

```diff
@@ -2,24 +2,24 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 import { DISPLAY_TYPE } from '../../containers/navbar/DisplayTypes';
-import { BUTTON_TYPE } from '../../containers/navbar/ButtonTypes';
+// import { BUTTON_TYPE } from '../../containers/navbar/ButtonTypes';
```

Comment:
Why are we commenting out these lines? It shouldn't show up on production because they're all hidden under the "application" feature flag anyway.

Reply:
yeah that's probably because the "application" feature flag was enabled in `nwhacks2019-dev` when you saw the sign in button